fhirpy/base/lib.py

```diff
@@ -452,20 +416,24 @@ def fetch_all(self, *, skip_caching=False):
 
         return resources
 
-    def get(self, id, *, skip_caching=False):
-        res_data = self.client._fetch_resource(
-            '{0}/{1}'.format(self.resource_type, id)
-        )
-
-        if res_data['resourceType'] != self.resource_type:
+    def get(self, id=None):
```

Comment:
https://github.com/beda-software/fhir-py/issues/29 
Let's allow any kwargs to get?

e.g. resources('Patient').get(id=1, active=True) is the same resources('Patient').search(id=1, active=True).get()

Reply:
As for me, it looks strange that we allow the only id field